endpoints/auction_test.go

```diff
@@ -274,6 +274,13 @@ func DummyPrebidCacheServer(w http.ResponseWriter, r *http.Request) {
 	w.Write(b)
```

Comment:
Please rollback the modifications made to this file. If you add the required  entries to `config/config.go`discussed in our comment above, these changes are unnecessary

Reply:
I'll revert the suggested files, thanks